app/src/main/java/org/cru/godtools/ui/profile/GlobalDashboardFragment.kt

```diff
@@ -4,13 +4,13 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.fragment.app.Fragment
 import androidx.fragment.app.viewModels
 import org.cru.godtools.databinding.FragmentGlobalDashboardBinding
-import org.cru.godtools.fragment.BasePlatformFragment
 import org.cru.godtools.viewmodel.GlobalDashboardDataModel
 import java.util.Calendar
 
-class GlobalDashboardFragment : BasePlatformFragment() {
+class GlobalDashboardFragment : Fragment() {
```

Comment:
why are you no longer extending `BasePlatformFragment`?

Reply:
I wasn't using anything from the BasePlatformFragment so I didn't use it.

Reply:
If you are planning on using data binding there is a `BaseBindingPlatformFragment` available to extend now.

Currently you only have access to the binding object when the binding is created or destroyed, I want to see if that provides everything we need when using databinding in a fragment to avoid needing to use a nullable property everywhere